Add deprecation warning to placeholder (#114)
equinor/webviz-core-components: CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [UNRELEASED] - YYYY-MM-DD
### Added

> - [#114](https://github.com/equinor/webviz-core-components/pull/114) - Added deprecation warning to WebvizPluginPlaceholder

Reviewer: ```suggestion
- [#114](https://github.com/equinor/webviz-core-components/pull/114) - Added deprecation warning to `WebvizPluginPlaceholder`.
```

### Changed

> - [#114](https://github.com/equinor/webviz-core-components/pull/114) - Better alignment of tooltips with icons and pointer cursor when hovering buttons in WebvizPluginPlaceholder

Reviewer: ```suggestion
- [#114](https://github.com/equinor/webviz-core-components/pull/114) - Better alignment of tooltips with icons and pointer cursor when hovering buttons in `WebvizPluginPlaceholder`.
```

### Fixed

> - [#114](https://github.com/equinor/webviz-core-components/pull/114) - Fixed bug in WebvizPluginPlaceholder preventing tooltips from being shown.

Reviewer: ```suggestion
- [#114](https://github.com/equinor/webviz-core-components/pull/114) - Fixed bug in `WebvizPluginPlaceholder` preventing tool tips from being shown.
```
